packages/collector/test/nativeModuleRetry/test.js

```diff
@@ -6,49 +6,19 @@
 'use strict';
 
 const { expect } = require('chai');
-const fsPromise = require('node:fs/promises');
-const os = require('os');
 const path = require('path');
 const fs = require('fs');
+const os = require('os');
 
 const config = require('../../../core/test/config');
 const { retry } = require('../../../core/test/test_util');
 const ProcessControls = require('../test_util/ProcessControls');
 const globalAgent = require('../globalAgent');
+const { execSync } = require('child_process');
 
-/**
- * Test suite for verifying the fallback mechanism for loading native add-ons.
- *
- * This test ensures that if a native add-on (such as event-loop-stats or gcstats.js) cannot be loaded initially,
- * the system correctly falls back to using precompiled versions of these add-ons. This ensures the application
- * functions as expected even if the native module is missing or cannot be compiled. The function `attemptRequire`,
- * defined in the shared-metrics/src/util/nativeModuleRetry  is used for the initial attempt to load the native
- * add-on module. It tries to require the native module from the root for this test, because we use npm workspaces
- * and the dependencies are hoisted to the root. If this attempt fails, the fallback mechanism is triggered.
- *
- * **Design problem of the test**:
- *
- * With `npm workspaces`, dependencies are installed in the root `node_modules` folder instead of individual
- * package `node_modules` folders. This can complicate managing the installed native modules and their copied
- * precompiled binaries, as the fallback mechanism must copy to the correct path. This test verifying that
- *  precompiled binaries are copied and loaded correctly to the @instana/shared-metrics/node_modules folder.
- * The test is **mis-designed**, because the test should not use the local dev environment to check the presence
- * of precompiled binaries. Instead it should create a tmp folder.
- *
- * **Test Design**:
- *
- * 1. **Setup**:
- *    - The installed native module is temporarily moved to a backup directory to simulate its absence. As the system
- *      first tries to require the native module from the root, so we relocate the modules.
- *
- * 2. **Fallback Mechanism**:
- *    - The test checks if the fallback mechanism correctly copies the precompiled binaries to the appropriate directory
- *      when the native module is not found.
- *
- * Note: Currently, we are not addressing the wrong test design because there will be a migration to a tool called
- *  prebuildify in the near future, which will change the whole feature. refs: INSTA-770
```

Comment:
Removed the todo from 770

> packages/collector/test/nativeModuleRetry/test.js
This test needs to be reworked or removed because we move our local dependencies  and test if prebuilds work. This is a bad approach - we always should install the collector in a tmp folder and then operate on this folder.
When moving to prebuildify, the prebuilds are copied in the installation step. We probably need a different test anyway.